### PR TITLE
Added the introductory content lab.

### PR DESCRIPTION
**Pull request by matejak**
I have added script for a lab track in form of an `.adoc` file.
The content has some gaps (it depends on yet-to-be-developed features and lacks screenshots), but the structure is ready for review. I recommend to go to the list of changed files and use the "View file" button to get the HTML from the .adoc source.

### CODE REVIEW

File: 2019Labs/CustomSecurityContent/documentation/lab0_introduction.adoc [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+
+image::0-01-setup.png[]
+
+=== Build the content
+
+The project consists of human-friendly files, that are compiled into standard-compliant files that are difficult to read and to edit directly.
+Therefore, let's see how the output looks, let's build the content!
+
+The https://github.com/ComplianceAsCode/content[ComplianceAsCode/content] project uses https://cmake.org/[cmake] build system.
+The build itself is based on Python, the `oscap` tool, and XSLT transforms.
```
Comment (jan-cerny) [inline 2019Labs/CustomSecurityContent/documentation/lab0_introduction.adoc]: transformations

File: 2019Labs/CustomSecurityContent/documentation/lab0_introduction.adoc [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+This name-coding is relevant for all types of additional content.
+
+Now, let's describe currently-supported associated content types:
+
+
+==== Checks
+
+Checks can be found under the `oval` directory.
+They are written in an standardized, declarative, XML-based language called OVAL (abbreviation of `Open Vulnerability and Assessment Language`).
+Writing checks in this language is considered cumbersome, and the `complianceAsCode` project helps it's users to write it more efficiently.
```
Comment (jan-cerny) [inline 2019Labs/CustomSecurityContent/documentation/lab0_introduction.adoc]: ComplianceAsCode

File: 2019Labs/CustomSecurityContent/documentation/lab0_introduction.adoc [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+
+We won't go into details of OVAL now, we just point out that the OVAL content can be found in a rule's subdirectory `oval`.
+If you are familiar with the language, you may take the opportunity to examine the `oval` subdirectory of the `accounts_tmout` rule's directory - there is the `shared.xml` file, that features a shorthand OVAL, which is much simpler than the full-bodied OVAL that you would have to write otherwise.
+
+
+==== Remediations
+
+Remediations can be found under `bash`, `ansible`, `anaconda` or `puppet` directories.
+If the system is not set up according to the rule’s description, the scanner reports an error, and the system administrator is supposed to fix it.
+The complianceAsCode content provides users with snippets that they can run and that can make the system compliant again, or that can provide administrators with hint of what they need to do.
```
Comment (jan-cerny) [inline 2019Labs/CustomSecurityContent/documentation/lab0_introduction.adoc]: ComplianceAsCode

File: 2019Labs/CustomSecurityContent/documentation/lab0_introduction.adoc [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+
+
+=== Customize a parametrized rule
+
+What if we want to have a shorter timeout than the OSPP policy requires?
+In the following section, we will learn about parametrized rules by taking following steps:
+
+. Learn where the value comes from.
+. Learn how is it applied to the rule.
+. Change it, and observe the result.
```
Comment (jan-cerny) [inline 2019Labs/CustomSecurityContent/documentation/lab0_introduction.adoc]: here you have 3 steps, but below you elaborate 4 steps

**Comment by matejak** [comment]
Self-merging, as the comments were addressed.

**Pull request merged**